jvb/src/test/java/org/jitsi/videobridge/xmpp/MediaSourceFactoryTest.java

```diff
@@ -109,25 +99,20 @@ public void createMediaSources1()
 
         MediaSourceDesc[] sources =
             MediaSourceFactory.createMediaSources(
-                Arrays.asList(videoSource, rtx), Arrays.asList(rtxGroup));
+                Arrays.asList(videoSource, rtx), Collections.singletonList(rtxGroup));
 
         assertNotNull(sources);
         assertEquals(1, sources.length);
         MediaSourceDesc source = sources[0];
         assertEquals(1, source.numRtpLayers());
     }
 
-    // 3 sim streams, 3 rtx -> 1 source, 3 encodings
+    // 3 sim streams, 3 rtx -> 1 source, 9 encodings
```

Comment:
Should say "layers" not "encodings"

Reply:
Any other places that should be changed?

Reply:
Should all uses of "encodings" be changed to "layers" in this file?

Reply:
Yes, probably, unless they're specifically referring to the encoded stream (i.e. all packets with a given ssrc).